lib/phoenix/controller.ex

```diff
@@ -206,16 +217,29 @@ defmodule Phoenix.Controller do
 
   @doc """
   Stores the view for rendering.
+
+  Raises `Phoenix.ControllerAlreadySentError` if the conn was already sent.
   """
   @spec put_view(Plug.Conn.t, atom) :: Plug.Conn.t
+  def put_view(%Plug.Conn{state: state}, _module) when not state in @unsent do
```

Comment:
What do you think about changing things around so that the guard would read `when state in @sent` instead? My brain gets hung up on the double negative.